src/utils/inputValidation/preloadedData.js

```diff
@@ -0,0 +1,56 @@
+import { isObject, isString } from '../lang';
+import logFactory from '../logger';
+const log = logFactory('', {
+  displayAllErrors: true
+});
+
+function validateSinceData(maybeSince, method) {
+  if (maybeSince > -1) return true;
+  log.error(`${method}: preloadedData.since must be a positive number.`);
+  return false;
+}
+
+function validateSplitsData(maybeSplitsData, method) {
+  if (isObject(maybeSplitsData)) {
+    const splitNames = Object.keys(maybeSplitsData);
+    if (splitNames.length > 0 && splitNames.every(splitName => isString(maybeSplitsData[splitName]))) return true;
+  }
+  log.error(`${method}: preloadedData.splitsData must be a map of split names to their serialized definitions.`);
```

Comment:
Maybe we should make this a separate validation. 
For an empty list we throw a warning (since it's possible to have an empty list of splits for an env).

For a list with items, if not all have the right format (keep isString for now or go and add the basic validation for Splits already, to check the key parts of the JSON) we'll log an error for invalid values.

Reply:
Thanks. It is a good idea to use the existing split validation utils